### PR TITLE
bears: Add puppet lint bear [WIP]

### PR DESCRIPTION
**Pull request by sudheesh001**
Fixes https://github.com/coala-analyzer/coala-bears/issues/46

### CODE REVIEW

File: bears/configfiles/PuppetLintBear.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    severity_map = {
+        'error': RESULT_SEVERITY.MAJOR,
+        'warning': RESULT_SEVERITY.NORMAL
+    }
+    use_stdout = True
+
+    def run(self, filename, file, puppet_cli_options: str=""):
+        '''
+        Checks the code with puppet-lint`. This will run puppet-lint
+        over all puppet files seperately.
```
Comment (tushar-rishav) [inline bears/configfiles/PuppetLintBear.py]: @sudheesh001  Shouldn't we mention about `puppet_cli_options` in docstrings?

Comment (sudheesh001) [inline bears/configfiles/PuppetLintBear.py]: This is a WIP, I will be doing this once the functionality is up, i've mentioned the problem i am facing on gitter

**Comment by sils1297** [comment]
any progress? Need help?

**Comment by Adrianzatreanu** [comment]
could you please run ``coala`` over your code? and a rebase :) that should fix it

**Comment by Adrianzatreanu** [comment]
@sudheesh001 you still here? its been 22 days since your last answer here, making sure you are okay :)

**Comment by sils1297** [comment]
@Adrianzatreanu time to take over?

**Comment by Adrianzatreanu** [comment]
none, got 2 exams in the following 2 days and a homework for wednesday, sorry.

**Comment by hypothesist** [comment]
> got 2 exams in the following 2 days and a homework for wednesday, sorry.

@Adrianzatreanu how about now? :P

**Comment by Adrianzatreanu** [comment]
this needs to be restarted, as its with the old linter API, ill take it if nobody does and if i have time :) however, its a cool bear so maybe a newcomer would want to take over it :)

**Comment by AbdealiJK** [comment]
Because this has been inactive for a while now I've taken this in at https://github.com/coala-analyzer/coala-bears/pull/747

**Pull request closed without merging**